apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -1,14 +1,120 @@
 // Pythonlab view
-import React from 'react';
-import PythonEditor from './PythonEditor';
+import React, {useEffect, useState} from 'react';
 import moduleStyles from './pythonlab-view.module.scss';
-import SidePanel from './SidePanel';
+import {ConfigType} from '@cdo/apps/weblab2/CDOIDE/types';
+import Editor from '../weblab2/CDOIDE/CenterPane/Editor';
+import {LanguageSupport} from '@codemirror/language';
+import {python} from '@codemirror/lang-python';
+import {CDOIDE} from '../weblab2/CDOIDE';
+import {MultiFileSource} from '../lab2/types';
+import Lab2Registry from '../lab2/Lab2Registry';
+import {useAppDispatch, useAppSelector} from '../util/reduxHooks';
+import {setSource} from './pythonlabRedux';
+import PythonConsole from './PythonConsole';
+
+const pythonlabLangMapping: {[key: string]: LanguageSupport} = {
+  py: python(),
+};
+
+const defaultProject: MultiFileSource = {
+  files: {
+    '0': {
+      id: '0',
+      name: 'main.py',
+      language: 'py',
+      contents: 'print("Hello world!")',
+      folderId: '1',
+      active: true,
+      open: true,
+    },
+  },
+  folders: {
+    '1': {
+      id: '1',
+      name: 'src',
+      parentId: '0',
+    },
+  },
+};
+
+const defaultConfig: ConfigType = {
+  //showSideBar: true,
```

Comment:
Did you leave these commented out as an indicator we'll want to change them?

Reply:
Mostly I copied this over from WebLab2View and kept the commented out pieces to indicate we haven't set them yet.

Reply:
I'll plan to clean these up as a follow-up